File: 0242-valid-anagram/0242-valid-anagram.py
class Solution(object):
    def isAnagram(self, s, t):
        s = sorted(s)
        t = sorted(t)
        n1 = len(s)
        n2 = len(t)

        if(n1 != n2):
            return False

        for i in range(n1):
            if(s[i] != t[i]):
                return False
        return True
        

File: 0242-valid-anagram/0242-valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        int n = s.size();
        int m = t.size();

        if(n != m) return false;
        int count[26] = {0};
        for(auto x: s){
            count[x-'a']++;
        }
        for(auto x:t){
            if(count[x-'a'] == 0)return false;
            else count[x-'a']--;
        }
        return true;
    }
};